### PR TITLE
Readme with details

### PR DESCRIPTION
**Pull request by bobjflong**

### CODE REVIEW

**Comment by eugeneius** [comment]
:book:

File: readme.md [hunk 1]
```diff
@@ -14,6 +14,22 @@ Running tests requires [phpunit](https://phpunit.de/).
...
+
+Once filled out, the Intercom widget will automatically appear:
+
+<img src="https://raw.githubusercontent.com/intercom/intercom-wordpress/master/screenshots/widget.png"/>
+
+# Users
+
+If a `$current_user` is present, we use their email as an identifier in the widget.
+
+Otherwise the widget operates in [Acquire mode](https://www.intercom.io/live-chat) (if available). This allows you to chat to anonymous users.
```
Comment (bscanlan) [inline readme.md]: s/to chat to anonymous users/to talk with anonymous visitors on your Wordpress site/ ? 

More similar to the marketing blurb here: https://www.intercom.io/live-chat

File: readme.txt [hunk 2]
```diff
@@ -0,0 +1,29 @@
+=== Plugin Name ===
+Contributors: bobintercom
+License: Apache 2.0
```
Comment (bscanlan) [inline readme.txt]: Maybe you should look into being a lawyer :)

**Comment by bscanlan** [comment]
:rocket:

**Pull request merged**